roadmap/2022.md

```diff
@@ -54,3 +54,34 @@ Misc:
   - [ ] Finish changes regarding the API keys
   - [ ] Finish changes regarding the new index routes
   - [ ] Add the tenant tokens methods
+
+## Q2
+
+Analytics:
+- [ ] Finish the implementation in the remaining SDKs (Java, Javascript, Rust)
+- [ ] Plan how we will implement the analytics on the non-sdk.
+- [ ] _(may not be required)_ Define the specs to support analytics in front-end tools with core team.
+- [ ] Implement it on the all the integrations maintained by the integration team.
+
+Processes:
+- [ ] Change the Integrations' Team developers scope: make it adapted to each developer, but also to reduce the context switches.
+
+Improvements:
+- [ ] Add a helpful docker/compose setup in every integration.
+- [ ] Code coverage:
+  - [ ] Define the tool which will be used (codecov, coveralls, others).
+  - [ ] Implement it in the SDKs.
+  - [ ] Implement it on the integrations which makes sense to have coverage.
+
+Meilisearch releases:
+- [ ] v0.27 work:
+  - [ ] Docker run changes
+  - [ ] Support nested fields
+  - [ ] Customize the typo tolerance
+  - [ ] Add support to use the new cropping and highlighting search attributes
+- [ ] v0.28 work _(to be completed when the final pre-release comes)_:
+  - [ ] New API namings, could impact multiple resources and nomenclatures, like dumps, keys, search, and indexes.
+  - [ ] New behavior of the keys expiresAt field.
+  - [ ] Improve understanding of the pagination of search results.
```

Comment:
Not on the integration team side (as far as I know). Mostly a work for docs team